File: Beginner/1036 - Bhaskara's Formula/main.py
aux = input()
name = str(aux)

line = name.strip().split(' ')
a = float(line[0])
b = float(line[1])
c = float(line[2])

if a == 0.0:
	print("Impossivel calcular")
else:
	root = (b * b) - (4 * a * c)

	if root < 0.0:
		print("Impossivel calcular")
	else:
		r1 = ((-b) + (root**(1/2))) / (2 * a)
		r2 = ((-b) - (root**(1/2))) / (2 * a)

		print("R1 = %.5f" % (r1))
		print("R2 = %.5f" % (r2))

File: Beginner/1036 - Bhaskara's Formula/main.cs
using System;
using System.Linq;

class URI
{
    static void Main(string[] args)
    { 
        double[] values = Array.ConvertAll(Console.ReadLine().Split(' '), double.Parse);
        double a = values[0];
        double b = values[1];
        double c = values[2];

        double root = Math.Pow(b, 2) - (4 * a * c);

        if (a == 0.0)
        	Console.WriteLine("Impossivel calcular");
        else if (root < 0.0)
        	Console.WriteLine("Impossivel calcular");
        else
        {
        	double r1 = ((-b + Math.Sqrt(root)) / (2 * a));
        	double r2 = ((-b - Math.Sqrt(root)) / (2 * a));
        	Console.WriteLine("R1 = " + r1.ToString("0.00000"));
        	Console.WriteLine("R2 = " + r2.ToString("0.00000"));
        }
    }
}